test/modeles/televersement/televersementServices.spec.js

```diff
@@ -104,4 +108,229 @@ describe('Un téléversement de services', () => {
+          depotDonnees.ajouteDossierCourantSiNecessaire = async (idService) => {
+            idRecu = idService;
+            return new Dossier({ id: 'D1' });
+          };
+
+          await televersement.creeLesServices(
+            'U1',
+            [],
+            depotDonnees,
+            busEvenement
+          );
+          expect(idRecu).to.be('S1');
+        });
+
+        it('remplis le dossier avec les données en utilisant les méthodes métier', async () => {
+          const dossierRecu = new Dossier({ id: 'D1' }, referentiel);
+          depotDonnees.ajouteDossierCourantSiNecessaire = async () =>
+            dossierRecu;
+
+          await televersement.creeLesServices(
+            'U1',
+            [],
+            depotDonnees,
+            busEvenement
+          );
+
+          expect(dossierRecu.toJSON()).to.eql({
+            id: 'D1',
+            finalise: true,
+            importe: true,
+            decision: {
+              dateHomologation: '01/01/2025',
+              dureeValidite: 'sixMois',
+            },
+            autorite: {
+              nom: 'Nom Prénom',
+              fonction: 'Fonction',
+            },
+            avecAvis: false,
+            avis: [],
+            avecDocuments: false,
+            documents: [],
+            dateTelechargement: {
+              date: '01/01/2025',
+            },
+          });
+        });
+
+        it("délègue au dépôt de données l'enregistrement du dossier finalisé", async () => {
+          let donneesRecues;
+          depotDonnees.enregistreDossier = async (idService, dossier) => {
+            donneesRecues = { idService, dossier };
+          };
+
+          await televersement.creeLesServices(
+            'U1',
+            [],
+            depotDonnees,
+            busEvenement
+          );
+
+          expect(donneesRecues.idService).to.be('S1');
+          expect(donneesRecues.dossier.finalise).to.be(true);
+          expect(donneesRecues.dossier.importe).to.be(true);
+        });
+      });
+
+      it("ne créé pas de dossier si le service téléversé n'en a pas", async () => {
+        let depotAppele = false;
+        depotDonnees.ajouteDossierCourantSiNecessaire = async () => {
+          depotAppele = true;
+        };
+
+        televersement = new TeleversementServices(
+          {
+            services: [
+              {
+                ...donneesServiceValide,
+                dateHomologation: undefined,
+                dureeHomologation: undefined,
+                nomAutoriteHomologation: undefined,
+                fonctionAutoriteHomologation: undefined,
+              },
+            ],
+          },
+          referentiel
+        );
+
+        await televersement.creeLesServices(
+          'U1',
+          [],
+          depotDonnees,
+          busEvenement
+        );
+        expect(depotAppele).to.be(false);
+      });
+
+      it("délègue au dépôt de données l'ajout d'une suggestion d'action", async () => {
```

Comment:
it("délègue au dépôt de données l'ajout d'une suggestion d'action invitant à compléter la description du service"